src/select_GCMs.R

```diff
@@ -18,24 +18,26 @@
 select_GCMs <-
   function(park,
            #region = "mean",
-           future_range = 2035:2065,
+           future_range = 2035:2065, 
            historic_range = 1979:2012,
            #low_skill_cutoff = 0.1,
            method = "pca",
            save = FALSE,
-           path = "data/") {
+           path = "data/park/") {
+    
     # READ IN CENTROID FILES --------
     # download and read in climate centroid files
     get_centroid_climate_data(park = park, save = FALSE)
     
     
     ## create list of future and historic dfs
-    ## MAKE SURE no other objects with '_future' or '_historical' in their names
+    ## MAKE SURE no other objects with 'PARKCODE_future' or 'PARKCODE_historical' in their names
+    # exist in the global environment.
     future_dfs <-
-      mget(ls(.GlobalEnv, pattern = "_future"), envir = .GlobalEnv)
+      mget(ls(.GlobalEnv, pattern = paste0(park, "_future")), envir = .GlobalEnv)
```

Comment:
This actually won't work since all koppen climate centroid files have the climate number in between the park name and "_future" etc. For example, a BRCA future koppen climate centroid data would be 'BRCA_07_future'. So this new method will only pull and perform climate selections for the park centroid and not the other koppen climate centroids (unless that is the intended use...but I think its safe to run it for all centroids). We will need to do some regrex for the `pattern = ` argument here, happy to help with that.

Reply:
Good point, we still might need those! I've updated the code so it pulls all centroids, not just park centroids.

Reply:
Okay I think this edit should work:
`mget(ls(.GlobalEnv, pattern = paste0(park, ".*?", "_future")), envir = .GlobalEnv)`